hnginternship5/ziki
# Added Profile Page

#### What does this PR do?
* It creates the profile page
#### Description of Task to be completed?
* Profile page `profile.html` is created
* Styling is added to `profile.css`
#### How should this be manually tested?
* Open the site on your browser
* Type localhost/profile
* You should see the profile page
#### Any background context you want to provide?
* I renamed the page to profile.html so I had to update the route in ```route.php```
#### What are the relevant pivotal tracker stories?
[#165477946](https://www.pivotaltracker.com/n/projects/2327725)
#### Screenshots (if appropriate)
#### Questions/Comments:
I couldn't add the image edit overlay effect
Also, gulp has been deleted so all the files point to the resources folder and not the build folder
```
